### PR TITLE
User Profile Verification

### PR DESCRIPTION
**Pull request by UmairMukhtar**
The account will be unverified to start with user will see button to send verification
unverified account
User will receive a link in the email
sentEmail to verify Account
clicking on the link the account will be verified
verified account

If the user will change the email in future the account will be converted to unverified account

### CODE REVIEW

**Comment by UmairMukhtar** [comment]
https://github.com/eLearningDAO/POCRE/issues/258

File: app/api/src/db/pool.ts [hunk 1]
```diff
@@ -55,6 +55,7 @@ const init = async (): Promise<QueryResult<any>> => {
       verified_id character varying,
       reputation_stars integer  DEFAULT 0,
       creation_count integer  DEFAULT 0,
+      email_verified integer  DEFAULT false,
```
[the remark this answers is not in the document]
Comment (UmairMukhtar) [reply app/api/src/db/pool.ts]: done thanks

[the remark this answers is not in the document]
Comment (AltiMario) [reply app/api/src/db/pool.ts]: 2 minor issues
image

[the remark this answers is not in the document]
Comment (AltiMario) [reply app/api/src/db/pool.ts]: ok, so I will test the email again after the merge, but please @UmairMukhtar look at the last screenshot with my comments

**Pull request merged**